ocaml/xapi/xapi_network.ml

```diff
@@ -20,23 +20,26 @@ open D
 open Db_filter
 open Network
 
-let create_internal_bridge ~__context ~bridge ~uuid =
+let create_internal_bridge ~__context ~bridge ~uuid ~persist =
 	debug "Creating internal bridge %s (uuid:%s)" bridge uuid;
 	let dbg = Context.string_of_task __context in
 	let current = Net.Bridge.get_all dbg () in
 	if not(List.mem bridge current) then
 		let other_config = ["network-uuids", uuid] in
-		Net.Bridge.create dbg ~name:bridge ~other_config ()
+		Net.Bridge.create dbg ~name:bridge ~other_config ();
+	Net.Bridge.set_persistent dbg ~name:bridge ~value:persist
```

Comment:
Good spot! 

> Or perhaps you knew this and the indentation is wrong?
Nope. I thought the if bound to a `;`... that was an oversight that `let` would gobble up forever.